back-end/sports_platform/sports_platform_api/views/user_views.py

```diff
@@ -53,3 +53,19 @@ def login(request):
     except Exception as e:
         return Response(data={"message": "Try later."}, status=500)
 
+@api_view(['POST'])
+def logout(request):
+    
+    if not request.user.is_authenticated:
+        return Response({"message": "User not logged in."},
+                        status=200)
```

Comment:
should we return 200 here? I am not sure

Reply:
I was not sure as well. I searched online and saw that this was recommended. Resources I found: https://stackoverflow.com/a/36220291/9789558
But now that I think about it, maybe it would be more clear if we return 401. 
What do you think?

Reply:
I do not have a clear idea, let's discuss this with the frontend and mobile team in today's meeting.